test/Constraints/members.swift

```diff
@@ -590,3 +590,36 @@ do {
   throw SR_2193_Error.Boom
 } catch let e as SR_2193_Error.Boom { // expected-error {{enum element 'Boom' is not a member type of 'SR_2193_Error'}}
 }
+
+// rdar://problem/25341015
+extension Sequence {
+  func r25341015_1() -> Int {
+    return max(1, 2) // expected-error {{type member 'max' shadows global function 'max' in module 'Swift', which cannot be called using implicit 'self.'}} {{12-12=Swift.}}
```

Comment:
I think mentioning "implicit self" doesn't help at all—it's clearly not what the user was intending to do if we get into this situation. Maybe something more like "use of 'max' refers to instance property 'max' rather than"

Actually, we were in this situation before with NSView.print(\_:) (5c71b75b250ef5d564a9bc7bc451fb72227de2fd). I think I like the message format we went with there:

```
<stdin>:5:5: warning: use of 'print' treated as a reference to instance method in class 'NSView'
    print("hi")
    ^
AppKit.NSView:7:15: note: 'print' declared here
    open func print(_ sender: Any?)
              ^
<stdin>:5:5: note: use 'self.' to silence this warning
    print("hi")
    ^
    self.
<stdin>:5:5: note: use 'Swift.' to reference the global function
    print("hi")
    ^
    Swift.
```